NEWS.md

```diff
@@ -16,6 +20,11 @@ rearrangements of Notcurses.
       `ncpile_create()`. The returned plane will be the top, bottom, and root
       of a new plane. Alternatively, use `ncplane_reparent()` or
       `ncplane_reparent_family()` with the source equal to the destination.
+    * Add new function `ncpile_render()`, which renders the pile containing the
+      specified plane to the specified buffer. Add new function
+      `ncpile_rasterize()` to rasterize the specified buffer to output.
+  * `notcurses_render_to_buffer()` has been removed, as there was no way to use
```

Comment:
NAK. We pledged to an API. Removing this reduces user trust. Put it back, asshole.

Reply:
Done.